### PR TITLE
applogs - AUT2-6

### PR DESCRIPTION
**Pull request by LayZ2001**
done it

### CODE REVIEW

**Review by jpat2398 — APPROVED** [review]

File: src/test/java/Reporting/logging.java
Comment (Zeeks17) [inline src/test/java/Reporting/logging.java:1]: is this extra page needed?

**Review by Zeeks17 — CHANGES_REQUESTED** [review]

**Review by Zeeks17 — APPROVED** [review]
looks good

**Review by JTAfo1 — APPROVED** [review]
Lovely stuff!

File: src/test/java/Reporting/logging.java
Comment (LayZ2001) [reply src/test/java/Reporting/logging.java:1]: deleted them now

**Review by j-forcer90 — APPROVED** [review]
should be ok - will let you resolve merge conflict

**Review by JackHDean21 — APPROVED** [review]

**Pull request merged**